diesel_cli/src/cli.rs

```diff
@@ -162,6 +166,16 @@ pub fn build_cli() -> App<'static, 'static> {
                 .possible_values(&Shell::variants()),
         );
 
+    let features_subcommand = SubCommand::with_name("features")
+        .about("list support for which database engines have been compiled into this version of diesel")
```

Comment:
Needs better phrasing of the description.

Reply:
I agree. Any suggestions?